src/LogEntry.ts

```diff
@@ -2,7 +2,10 @@ import {LogLevel} from "./LogLevel";
 
 export interface LogEntry {
     level: LogLevel;
+    object: any;
```

Comment:
Separating text and object is a good idea, but I insist on passing object data as string, not as object itself. It would give some overhead in JSON serializing/unserializing, but only in console appender